layout.coffee

```diff
@@ -1189,13 +1197,36 @@ window.dr = do ->
       @css_baseclass = 'sprite'
       @_updateClass()
 
-    setStyle: (name, value, internal, el=@el) ->
-      value ?= ''
-      name = stylemap[name] if name of stylemap
-      value = styleval[name](value) if name of styleval
-      # console.log('setStyle', name, value, @el)
-      el.style[name] = value
-      # @jqel.css(name, value)
+    setStyle: (() ->
```

Comment:
Perhaps you want this: http://stackoverflow.com/questions/11869270/coffeescript-do-pass-argument